examples/index.tsx

```diff
@@ -2,258 +2,1083 @@
-export { default as SliderTooltipCustomExample } from './Slider.tooltipcustom';
-export { default as SliderTooltipNoneExample } from './Slider.tooltipnone';
-
-export { default as StepperMainExample } from './Stepper.main';
-export { default as StepperShortExample } from './Stepper.short';
-export { default as StepperLongExample } from './Stepper.long';
-export { default as StepperTooltipExample } from './Stepper.tooltip';
-export { default as StepperLocalizationExample } from './Stepper.localization';
-export { default as StepperLayoutExample } from './Stepper.layout';
-
-export { default as SurfaceMainExample } from './Surface.main';
-export { default as SurfaceElevationExample } from './Surface.elevation';
-export { default as SurfaceHeaderFooterExample } from './Surface.headerfooter';
-export { default as SurfaceNoPaddingExample } from './Surface.nopadding';
-
-export { default as TableMainExample } from './Table.main';
-
-export { default as TabsMainExample } from './Tabs.main';
-export { default as TabsOverflowExample } from './Tabs.overflow';
-
-export { default as TagMainExample } from './Tag.main';
-export { default as TagBasicExample } from './Tag.basic';
-export { default as TagDefaultExample } from './Tag.default';
-
-export { default as TagContainerMainExample } from './TagContainer.main';
-export { default as TagContainerWrapExample } from './TagContainer.wrap';
-export { default as TagContainerTruncateExample } from './TagContainer.truncate';
-export { default as TagContainerScrollExample } from './TagContainer.scroll';
-
-export { default as TextareaMainExample } from './Textarea.main';
-export { default as TextareaScrollExample } from './Textarea.scroll';
-export { default as TextareaStatusExample } from './Textarea.status';
-export { default as TextareaInlineExample } from './Textarea.inline';
-
-export { default as TextMainExample } from './Text.main';
-
-export { default as TileMainExample } from './Tile.main';
-
-export { default as ToastMainExample } from './Toast.main';
-
-export { default as ToggleSwitchMainExample } from './ToggleSwitch.main';
-export { default as ToggleSwitchSizesExample } from './ToggleSwitch.sizes';
-export { default as ToggleSwitchLabelsExample } from './ToggleSwitch.labels';
-export { default as ToggleSwitchIconExample } from './ToggleSwitch.icon';
-export { default as ToggleSwitchInputGroupExample } from './ToggleSwitch.inputgroup';
-
-export { default as TooltipMainExample } from './Tooltip.main';
-export { default as TooltipPlacementExample } from './Tooltip.placement';
-
-export { default as TransferListMainExample } from './TransferList.main';
-export { default as TransferListWithLabelExample } from './TransferList.withlabel';
-
-export { default as TreeMainExample } from './Tree.main';
-
-export { default as VisuallyHiddenIconExample } from './VisuallyHidden.icon';
-export { default as VisuallyHiddenMoreTextExample } from './VisuallyHidden.moretext';
+import * as React from 'react';
```

Comment:
That's a lot of lines 👀

Reply:
yeah, but it's a lot fewer lines than adding `<ThemeProvider></ThemeProvider>` manually to every single example usage - especially when you consider that we will start using examples in other places (like #1352)